infra/op-replica/docker-compose/README.md

```diff
@@ -20,74 +18,86 @@ The `data-transport-layer` should run with 1 CPU and 256Mb of memory.
 
-!! Update DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT to a valid endpoint !!
+   ```sh
+   cp default-mainnet.env .env
+   ```
 
 ### Settings
 
+Edit the settings in `.env`.
+
+```sh
+nano .env
+```
+  
+   
 Change any other settings required for your environment
 
 | Variable                 | Purpose                                                  | Default
 | ------------------------ | -------------------------------------------------------- | -----------
 | COMPOSE_FILE             | The yml files to use with docker-compose                 | replica.yml:replica-shared.yml
-| ETH_NETWORK              | Ethereum Layer1 and Layer2 network (mainnet,kovan)       | kovan (change to `mainnet` for the production network)
-| DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT | An endpoint for the L1 network, either kovan or mainnet.
-| DATA_TRANSPORT_LAYER__L2_RPC_ENDPOINT | Optimistic endpoint, such as https://kovan.optimism.io or https://mainnet.optimism.io
+| ETH_NETWORK              | Ethereum Layer1 and Layer2 network (mainnet,kovan)       | Depends on the configuration file you use
+| DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT | An endpoint for the L1 network, either kovan or mainnet. You **must** change this to a valid value.
+| DATA_TRANSPORT_LAYER__L2_RPC_ENDPOINT | [Optimistic endpoint](https://community.optimism.io/docs/useful-tools/networks/), such as https://kovan.optimism.io or https://mainnet.optimism.io
 | REPLICA_HEALTHCHECK__ETH_NETWORK_RPC_PROVIDER | The L2 endpoint to check the replica against | (typically the same as the DATA_TRANSPORT_LAYER__L2_RPC_ENDPOINT)
 | SEQUENCER_CLIENT_HTTP | The L2 sequencer to forward tx to  | (typically the same as the DATA_TRANSPORT_LAYER__L2_RPC_ENDPOINT)
-| SHARED_ENV_PATH          | Path to a directory containing env files                 | [a directory under ./kustomize/replica/envs](https://github.com/optimisticben/op-replica/tree/main/kustomize/replica/envs)
-| GCMODE                   | Whether to run l2geth as an `archive` or `full` node     | archive
-| L2GETH_IMAGE_TAG         | L2geth version                                           | 0.5.8 (see below)
-| DTL_IMAGE_TAG            | Data transport layer version                             | latest (see below)
-| HC_IMAGE_TAG             | Health check version                                     | latest (see below)
+| SHARED_ENV_PATH          | Path to a directory containing env files                 | [a directory under .../op-replica](https://github.com/ethereum-optimism/optimism/tree/develop/infra/op-replica/envs)
+| GCMODE                   | [Whether to run l2geth as an `archive` or `full` node](https://www.quicknode.com/guides/infrastructure/ethereum-full-node-vs-archive-node)     | archive
+| L2GETH_IMAGE_TAG         | L2geth version                                           | [Go here](https://hub.docker.com/r/ethereumoptimism/l2geth/tags) and find the latest version. At writing this is 0.5.19 (1).
```

Comment:
I really don't like the idea of setting a manual tag version in the README. It updates too often and this change is missed.

We should have a way to publish the current versions supported and let user's choose from the list.

This README should point to that list.